src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -833,6 +855,8 @@ void makeNotNullMembersMaybeNull()
                             var memberToInitialize = member;
                             switch (member)
                             {
+                                case PropertySymbol { IsRequired: true }:
```

Comment:
Comments here which refer to manually implemented properties should perhaps be updated to refer to manually implemented, non-required properties.